asset/css/markbind.css

```diff
@@ -202,3 +202,13 @@ footer {
     0% { transform: rotate(0deg); }
     100% { transform: rotate(360deg); }
 }
+
+/* Temporary Styles */
```

Comment:
We need to explain the rationale for having and using temporary styles (which is to avoid the FOUC problem).
I think this should be the best place to put the entire explanation in.

Reply:
Then perhaps we can also name all references as "Temporary **Anti-FOUC** Styles" rather than just "Temporary Styles"? This may make it even more clearer to the developers how these temporary styles are used.

Reply:
Fixed!